.github/workflows/sdk-pr.yml

```diff
@@ -16,7 +16,7 @@ concurrency:
 
 jobs:
   formatting:
-    name: Assert code formattings
+    name: Assert code style
```

Comment:
Why this name change?

Reply:
Cause I don't know what went into my head when I invented the word "formattings"

Reply:
:-). As the job is called formatting, I would say Assert code formatting